packages/@react-spectrum/s2/src/index.ts

```diff
@@ -81,6 +81,7 @@ export {TreeView, TreeViewItem, TreeViewItemContent} from './TreeView';
 
 export {pressScale} from './pressScale';
 
+export {Autocomplete, AutocompleteContext, AutocompleteStateContext} from 'react-aria-components';
```

Comment:
do we need the context and statecontext exposed in s2?

Reply:
I think it makes sense to export everything for it that we do for RAC, but open to changing this.